services/src/main/java/org/keycloak/protocol/oidc/mappers/ScriptBasedOIDCProtocolMapper.java

```diff
@@ -120,7 +120,7 @@ public String getHelpText() {
 
   @Override
   public boolean isSupported() {
-    return Profile.isFeatureEnabled(Profile.Feature.SCRIPTS) && Profile.isFeatureEnabled(Profile.Feature.UPLOAD_SCRIPTS);
+    return Profile.isFeatureEnabled(Profile.Feature.SCRIPTS);
```

Comment:
Same question like for `ScriptBasedAuthenticator` .

Reply:
The provider is no longer registered as a service, that is why I did not change here.

Reply:
Yes, true. Thanks for pointing this!